nums/core/systems/interfaces.py

```diff
@@ -136,7 +136,13 @@ def svd(self, arr, syskwargs: Dict):
     def inv(self, arr, syskwargs: Dict):
         raise NotImplementedError()
 
-    def allclose(self, a, b, rtol, atol, syskwargs: Dict):
+    def array_equal(self, a, b, args, syskwargs: Dict):
```

Comment:
While it may make sense to break these out into separate functions in the future, for now can we consolidate these calls to a single function? Similar to how `array_eq` is implemented in `ArrayApplication`.

Reply:
Sounds good.